server/index.js

```diff
@@ -2,13 +2,38 @@ require('dotenv/config');
 const express = require('express');
 const staticMiddleware = require('./static-middleware');
 const errorMiddleware = require('./error-middleware');
-
+const jsonMiddleware = express.json();
+// const ClientError = require('./client-error');
+const pg = require('pg');
 const app = express();
 
 app.use(staticMiddleware);
+app.use(jsonMiddleware);
+
+const db = new pg.Pool({ // eslint-disable-line
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
 
-app.get('/api/hello', (req, res) => {
-  res.json({ hello: 'world' });
+app.get('/api/products', (req, res, next) => {
+  const sql = `
+  select "product_id",
+          "name",
+          "manufacturer",
+          "description",
+          "price",
+          "image_url",
```

Comment:
This means you're going to have to use `image_url` as a field name in your JavaScript. Ugly! You could rename the field in the schema and regenerate the database, or you could use `image_url as imageUrl`.